compiler/src/iree/compiler/Codegen/Dialect/GPU/TargetUtils/KnownTargets.cpp

```diff
@@ -142,6 +142,9 @@ const WgpDetails *getCDNA3WgpDetails() {
       MMAIntrinsic::MFMA_F32_16x16x32_F8E5M2FNUZ,
       MMAIntrinsic::MFMA_I32_16x16x32_I8,
       MMAIntrinsic::MFMA_I32_32x32x16_I8,
+      // Non-native MFMA intrinsics with unrolled_k = 2.
+      MMAIntrinsic::VMFMA_F32_16x16x32_F16,
+      MMAIntrinsic::VMFMA_F32_32x32x16_F16,
```

Comment:
Rather than specifying this as a part of the target info, we can expand the list of intrinsics on the fly in KernelConfig (supporting `MFMA_F32_16x16x16_F16` => support for `VMFMA_F32_16x16x32_F16`).

Reply:
That makes sense! Maybe we should somehow add a map that goes from native MFMA -> list of virtual MFMAs? Otherwise, I think it may look a bit ad-hoc in KernelConfig to do `if (intrinsic == MFMA_F32_16x16x16_F16) push_back( VMFMA_F32_16x16x32_F16)`. Any thoughts? :)

Reply:
I was thinking we can do something similar to `getASingleSubgroupLayout` we can have a classDeclaration on `IREEGPU_MMAAttr` to do `getVirtualIntrinsics()`

Reply:
Done, I think, LMK if you still have some inputs on it! :)